test/content.test.js

```diff
@@ -47,6 +47,26 @@ describe('Profile', () => {
       expect(profileSet.destProfiles[2]).to.deep.include({ profile: 'target2', role_name: 'DefaultRole' });
       expect(profileSet.excludedNames[0]).to.eq('target4');
     })
+
+    it('loads the configuration that contains base account with target_region', () => {
+      let profileSet = new ProfileSet([
+        { profile: 'target1', aws_account_id: '111122223334',
+          role_name: 'Role1', source_profile: 'base1' },
+        { profile: 'target2', aws_account_id: '111122223335',
+          source_profile: 'base1' },
+        { profile: 'base1', aws_account_id: '111100003333', target_region: 'DefaultRegion' },
+        { profile: 'base2', aws_account_id: '222200001111' },
+        { profile: 'targetex', aws_account_id: '333300001112',
+          role_name: 'roleex' },
+        { profile: 'target4', aws_account_id: '222200001112',
+          role_name: 'role3', source_profile: 'base2' }
+      ]);
+
+      expect(profileSet.destProfiles[0].profile).to.eq('targetex');
+      expect(profileSet.destProfiles[1]).to.deep.include({ profile: 'target1', region: 'DefaultRegion' });
+      expect(profileSet.destProfiles[2]).to.deep.include({ profile: 'target2', region: 'DefaultRegion' });
```

Comment:
This region should be the individual value after the above change was applied.

Reply:
Updated accordingly.